### DISCUSSION TITLE
Bonito Runtime for PromethION run

### DISCUSSION THREAD

**Discussion by ktan8** [nanoporetech/bonito#130] [Q&A]
I was wondering if anyone knows how long it'll take to complete a Bonito on a single PromethION run (E.g. on a V100 GPU)? Also, will there be any issues with running Bonito on a preemptible GPU VM instance on Google Engine (or is a standard instance preferred)?

**Comment by caspargross** [comment]
Very rough estimate: 1day/flowcell on the promethion with 2 V100 and 80 Threads.
